suites/nautilus/ansible/tier-2_deploy_test-ceph-rpms.yaml

```diff
@@ -26,6 +26,7 @@ tests:
       module: test_ansible.py
       config:
         is_mixed_lvm_configs: True
+        custom_log: true # CEPH-83572701
```

Comment:
Please modify the `polarion-id` key instead of adding a comment.

Reply:
@psathyan, previously, I added another test and it looked duplicate to this, hence @sunilangadi2 has suggested adding this custom_log config parameter to this test case and comment the polarion ID. To apply your suggestion, we'll need to add the two polarion ids in the polarion-id key!

Reply:
Automation requires that we should be able to pinpoint the test case that has failed. I believe this requirement is not met in this case hence the comment. Please discuss this in the FG

Reply:
@psathyan  Thanks, I'll discuss it more within FG. To get more information on it! Can we add two or more polarion IDs to `polarion-id`?

Reply:
If we add more than one Polarion ID... we would not be able to pin point which test case has failed.

Reply:
@psathyan, Thanks, I get your point and I quickly discussed this within the FG, as per the discussion, we do not want to create a duplication of the test and hence need to update the polarion ID within this test! Please help me if you have any other suggestions for the same.

Reply:
@psathyan: @manasagowri suggested to edit the existing test case `CEPH-83571467`  itself along with log validation step, even I like that idea, otherwise for polarion id purpose if we are going to create new test suite then it's increases in the pipeline execution time with all these new suites.
That is ok right @psathyan ?
what's your thoughts?

Reply:
@sunilangadi2 yes, it is one of the options we can take.